x-pack/test/cloud_security_posture_functional/page_objects/add_cis_integration_form_page.ts

```diff
@@ -175,12 +175,14 @@ export function AddCisIntegrationFormPageProvider({
     await integrationList[0].click();
   };
 
-  const clickLaunchAndGetCurrentUrl = async (buttonId: string, tabNumber: number) => {
+  const clickLaunchAndGetCurrentUrl = async (buttonId: string) => {
     const button = await testSubjects.find(buttonId);
     await button.click();
-    await browser.switchTab(tabNumber);
+    await new Promise((r) => setTimeout(r, 3000));
```

Comment:
Could you please explain why we need sleep? There are clear instructions for avoiding the use of sleep in tests.

Reply:
woops, good catch, I forgot to remove this
Originally I added sleep because I thought the steps was going too fast